src/fps/example/mainnetConfig.toml

```diff
@@ -1,4 +1,2 @@
 # this is the file used to determine the network configuration
-safeNonce = 67
```

Comment:
Why do we have this file in /example, same question for taskConfig.toml? Is it used? Can we delete it?